modules/RoutingContext.js

```diff
@@ -5,15 +6,28 @@ import getRouteParams from './getRouteParams'
 
 const { array, func, object } = React.PropTypes
 
-/**
- * A <RoutingContext> renders the component tree for a given router state
- * and sets the history object and the current location in context.
- */
 class RoutingContext extends Component {
 
   getChildContext() {
     const { history, location } = this.props
-    return { history, location }
+    const router = {
```

Comment:
Probably 5% to 10% as long as it takes for the GC to clean up all of the replaced `router` objects :p

But yeah it's probably a pointless micro-optimization.